@xen-orchestra/backups/_runners/_vmRunners/_Abstract.mjs

```diff
@@ -72,6 +73,11 @@ export const Abstract = class AbstractVmBackupRunner {
       return
     }
 
+    if (!this._hasTransferredData) {
```

Comment:
I think you could raise an error if this is undefined , and then do the this check